### PR TITLE
0.1.2 release

### PR DESCRIPTION
**Pull request by ted-miller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,23 +153,23 @@ To calculate the MD5 hash on Debian/Ubuntu for the main MotoROS2 binary, run the
...
-| YRC1000micro  | Humble           | `mr2_yrc1m_h.out` | `0.1.1`    | `804fd4d1e9f7b93835011203d8252880` |
+| DX200         | Foxy             | `mr2_dx2_f.out`   | `0.1.2`    | `9a8db7cd1ad436d6ace2cca8e0e2eb6b` |
+| DX200         | Galactic         | `mr2_dx2_g.out`   | `0.1.2`    | `d7e435ebf58c6b6da59555b941487af6` |
+| DX200         | Humble           | `mr2_dx2_h.out`   | `0.1.2`    | `95da6b0b93407855b8c133ece9a952be` |
+| YRC1000       | Foxy             | `mr2_yrc1_f.out`  | `0.1.2`    | `813e6ee6a72626bafffa54641d9b77f8` |
+| YRC1000       | Galactic         | `mr2_yrc1_g.out`  | `0.1.2`    | `41b249ef1132bce67284c41a68289e56` |
+| YRC1000       | Humble           | `mr2_yrc1_h.out`  | `0.1.2`    | `767c4f8ccc98eb4ee85146342c2984e7` |
+| YRC1000micro  | Foxy             | `mr2_yrc1m_f.out` | `0.1.2`    | `c1ca5fa9f44cf1e1e7de02174b3a6565` |
+| YRC1000micro  | Galactic         | `mr2_yrc1m_g.out` | `0.1.2`    | `20a6a332dc76bf657684a634b33c5c5e` |
+| YRC1000micro  | Humble           | `mr2_yrc1m_h.out` | `0.1.2`    | `22522443553457d5bd9ad08785d117ed` |
```
Comment (gavanderhoorn) [inline README.md:172]: You have the `.out`s already built?

**Review by gavanderhoorn — APPROVED** [review]
If you create a draft release, I can review that as well.

File: README.md (hunk 1 above)
Comment (ted-miller) [reply README.md:172]: Yes. I've been using the process written up [here](https://github.com/pyumr/motoros2-internal/discussions/633)

**Comment by ted-miller** [comment]
The draft release has been posted

**Pull request merged**